packages/integration-sdk-core/src/types/logger.ts

```diff
@@ -40,6 +40,10 @@ export type IntegrationEvent = {
 export type PublishEventInput = {
   name: string;
   description: string;
+  /**
+   * Allows the event to be identified via a code.
+   */
+  eventCode?: string;
```

Comment:
thought? But i think `eventCode` is chill as well.
```suggestion
  eventMetadata?: { [k: string]: string };
```

Reply:
This might cause issues with our consumer...